packages/block-library/src/embed/core-embeds.js

```diff
@@ -21,255 +21,257 @@ import {
+			description: __('Embed Cloudup content.'),
 		},
-		patterns: [ /^https?:\/\/cloudup\.com\/.+/i ],
+		patterns: [/^https?:\/\/cloudup\.com\/.+/i],
 	},
 	{
 		name: 'core-embed/collegehumor',
 		settings: {
 			title: 'CollegeHumor',
 			icon: embedVideoIcon,
-			description: __( 'Embed CollegeHumor content.' ),
+			description: __('Embed CollegeHumor content.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?collegehumor\.com\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?collegehumor\.com\/.+/i],
 	},
 	{
 		name: 'core-embed/dailymotion',
 		settings: {
 			title: 'Dailymotion',
 			icon: embedVideoIcon,
-			description: __( 'Embed a Dailymotion video.' ),
+			description: __('Embed a Dailymotion video.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?dailymotion\.com\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?dailymotion\.com\/.+/i],
 	},
 	{
 		name: 'core-embed/hulu',
 		settings: {
 			title: 'Hulu',
 			icon: embedVideoIcon,
-			description: __( 'Embed Hulu content.' ),
+			description: __('Embed Hulu content.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?hulu\.com\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?hulu\.com\/.+/i],
 	},
 	{
 		name: 'core-embed/imgur',
 		settings: {
 			title: 'Imgur',
 			icon: embedPhotoIcon,
-			description: __( 'Embed Imgur content.' ),
+			description: __('Embed Imgur content.'),
 		},
-		patterns: [ /^https?:\/\/(.+\.)?imgur\.com\/.+/i ],
+		patterns: [/^https?:\/\/(.+\.)?imgur\.com\/.+/i],
 	},
 	{
 		name: 'core-embed/issuu',
 		settings: {
 			title: 'Issuu',
 			icon: embedContentIcon,
-			description: __( 'Embed Issuu content.' ),
+			description: __('Embed Issuu content.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?issuu\.com\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?issuu\.com\/.+/i],
 	},
 	{
 		name: 'core-embed/kickstarter',
 		settings: {
 			title: 'Kickstarter',
 			icon: embedContentIcon,
-			description: __( 'Embed Kickstarter content.' ),
+			description: __('Embed Kickstarter content.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?kickstarter\.com\/.+/i, /^https?:\/\/kck\.st\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?kickstarter\.com\/.+/i, /^https?:\/\/kck\.st\/.+/i],
 	},
 	{
 		name: 'core-embed/meetup-com',
 		settings: {
 			title: 'Meetup.com',
 			icon: embedContentIcon,
-			description: __( 'Embed Meetup.com content.' ),
+			description: __('Embed Meetup.com content.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?meetu(\.ps|p\.com)\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?meetu(\.ps|p\.com)\/.+/i],
 	},
 	{
 		name: 'core-embed/mixcloud',
 		settings: {
 			title: 'Mixcloud',
 			icon: embedAudioIcon,
-			keywords: [ __( 'music' ), __( 'audio' ) ],
-			description: __( 'Embed Mixcloud content.' ),
+			keywords: [__('music'), __('audio')],
+			description: __('Embed Mixcloud content.'),
 		},
-		patterns: [ /^https?:\/\/(www\.)?mixcloud\.com\/.+/i ],
+		patterns: [/^https?:\/\/(www\.)?mixcloud\.com\/.+/i],
 	},
 	{
-		name: 'core-embed/polldaddy',
+		name: 'core-embed/crowdsignal',
```

Comment:
You can't only change the name of the block. You have to deprecate the old one and create a new and add a transform function. Otherwise the existing polldaddy blocks are broken. See the old `core-embed/speaker` and the new `core-embed/speaker-deck` embed.

Reply:
OIC, I didn't think to check existing Polldaddy blocks. Thanks for the education. I'll update.